### PR TITLE
Update webmessages_pl.properties

### PR DESCRIPTION
**Pull request by Waterrail**

### CODE REVIEW

File: assets/website/lang/webmessages_pl.properties [hunk 1]
```diff
@@ -82,9 +82,9 @@ modal_body = "Jeżeli nie pamiętasz danych logowania swojego starego konta - ni
 modal_button_register = "Zarejestruj nowe konto"
 
 [dashboard]
-name = "Dashboard"
+name = "Kokpit"
```
Comment (skmedix) [inline assets/website/lang/webmessages_pl.properties]: Dashboard is rarely translated into polish afaik

File: assets/website/lang/webmessages_pl.properties [hunk 2]
```diff
@@ -173,25 +173,25 @@ user_welcome = "Witaj,"
...
+button_back = "Powrót do strony głównej"
 
 [register_mail]
-subject = "Register Confirmation"
+subject = "Potwierdzenie rejestracji"
 
-welcome = "Hi"
-text = "thank you for registering your account on SKplatform."
-button_confirm = "Confirm My Email Address"
+welcome = "Cześć,"
```
Comment (skmedix) [inline assets/website/lang/webmessages_pl.properties]: Unnecessary comma as it's used like `Hi {nickname},`

Comment (Waterrail) [reply assets/website/lang/webmessages_pl.properties]: oh ok

File: assets/website/lang/webmessages_pl.properties (hunk 1 above)
Comment (Waterrail) [reply assets/website/lang/webmessages_pl.properties]: But I don't think I should leave it untranslated.

**Pull request merged**